Added some screenshots, steps and a comment about file formats. (#1117)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mule-app-properties-to-configure.adoc

http.host=my-api.cloudhub.io
----

> MuleSoft recommends using `.yaml` files because it supports the addition of type validations and autocompletion.

Reviewer: Per style ref. "file types and file extensions":
```suggestion
MuleSoft recommends using YAML files, to support the addition of type validations and autocompletion.
```

== Create and Configure your Properties File

> . In Studio, go to the *Package Explorer* view and right click the `/src/main/resources` folder in your project. Then, select *New* > *File*:

Reviewer: This is a lot of user activity for just one step. How about...:
```suggestion
. In Studio, go to the *Package Explorer* view and right-click the `/src/main/resources` folder in your project. 
. Select *New* > *File*:
```
Note also hyphenation of "right-click."

+
image:mruntime-properties-new-file.png[new-file-properties]

> . Choose a name for your file and set the extension to either `.yaml` or `.properties`, depending on which format you prefer to use.

Reviewer 2: as mentioned https://github.com/mulesoft/docs-mule-runtime/blob/v4.2/modules/ROOT/pages/configuring-properties.adoc#supported-files, maybe add the "disclaimer": 
> the recommended approach is to use a YAML configuration files
